.husky/.gitignore

```diff
@@ -1 +1 @@
-_
```

Comment:
This file should be kept as is, we do not push husky's script (`.husky/_/husky.sh`) to the repository.